@here/harp-mapview/lib/MapView.ts

```diff
@@ -614,6 +617,35 @@ export interface MapViewOptions {
      * @default false.
      */
     synchronousRendering?: boolean;
+
+    /**
+     * Set initial camera target in geo coordinates.
+     *
+     * Longitude values outside of -180 and +180 are acceptable.
+     * @default new GeoCoordinates(25, 0, 30000000)
+     */
+    target?: GeoPointLike | GeoCoordinatesLike | LatLngLike;
```

Comment:
Maybe add a dedicated type for that. Not sure how to name it though (maybe GeoCoordLike)